policies/broken-nightlies.md

```diff
@@ -0,0 +1,30 @@
+# Policy on broken nightlies
+
+Sometimes the nightlies released automatically by our CI ends up being broken
+for some people or even everyone. This policy defines what the infra team
+response will be in those cases.
+
+## Which nightly will be rolled back
+
+* A nightly can be rolled back if it contains destructive code, for
+  example if the included compiler deletes all the users files.
+
+* A nightly can be rolled back if an infra problem caused it to be broken for a
+  big percentage of users on any Tier 1 platform. Issues affecting only lower
+  tier platforms are not worthy of a roll back, since we don't guarantee working
+  builds for those platforms anyway.
+
+* A nightly will **not** rolled back if it contains a compiler bug, since those
+  bugs are expected to be caught by CI.
```

Comment:
I think I understand what you mean, but perhaps it could be clearer.

Reply:
Rewrote that paragraph, is it clearer?

Reply:
Yup!  Thanks.